docs/OnlineTutorial/guide.md

```diff
@@ -1293,26 +1293,26 @@ method ComputeFib(n: nat) returns (b: nat)
 ``` {.editonly}
 function fib(n: nat): nat
 {
-   if n == 0 then 0 else
-   if n == 1 then 1 else
+  if n == 0 then 0 else
+  if n == 1 then 1 else
                   fib(n - 1) + fib(n - 2)
```

Comment:
```suggestion
  if n == 0 then 0 else
  if n == 1 then 1 else
  fib(n - 1) + fib(n - 2)
```

Reply:
I think the value should have some indenting -- it is not a parallel statement to the if

Reply:
I consider this code similarly to:

```dafny
if n == 0 { return 0 }
if n == 1 { return 1}
return fib(n-1) + fib(n-2);
```
so that's why I wouldn't put indentation even on the last one.

Reply:
I agree with your indentation with return statements, but I disagree that it is similar to the example in question.
My thought is that items at the same level of indentation should be of the same grammatical sort -- e.g. a sequence of statements. In
```
if n == 1 then 1 else
fib(n - 1) + fib(n - 2)
```
the second line is a subexpression of the first, not something that is executed when the first statement is completed. At minimum it should be formatted as
```
if n == 1 then 1
else fib(n - 1) + fib(n - 2)
```
since then the `else` triggers ones mental parsing. But without the else, I still argue the expression on the second line should be indented.